CHANGELOG.md

```diff
@@ -12,8 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 ### Added
+
 - Add quickstart guide for background blur feature.
 - Add `BackgroundBlurCheckbox` component to allow selecting background blur video filter for the `PreviewVideo` component.
+- Add simple test demo App for integration testing.
```

Comment:
We do not need CHANGELOG for this right?

Reply:
If the CHANGELOG is only for SDK related changes, I think I can remove it.